### PR TITLE
Add missing get_width() and set_width() functions to WeightedDegreeRBFKernel

### PR DESCRIPTION
**Pull request by lkuchenb**

### CODE REVIEW

File: src/shogun/kernel/WeightedDegreeRBFKernel.h [hunk 1]
```diff
@@ -77,6 +77,19 @@ class CWeightedDegreeRBFKernel: public CDotKernel
 		virtual EFeatureType get_feature_type() { return F_DREAL; }
 
 
+		/** set width
```
Comment (karlnapf) [inline src/shogun/kernel/WeightedDegreeRBFKernel.h]: Minor: Capital "S"

File: src/shogun/kernel/WeightedDegreeRBFKernel.h [hunk 2]
```diff
@@ -77,6 +77,19 @@ class CWeightedDegreeRBFKernel: public CDotKernel
...
+		 *
+		 * @param w new width
+		 * @return if setting was successful
+		 */
+		inline bool set_width(float64_t w) { width=w; return true; }
```
Comment (karlnapf) [inline src/shogun/kernel/WeightedDegreeRBFKernel.h]: no need for inline, and also no need for a bool return

File: src/shogun/kernel/WeightedDegreeRBFKernel.h [hunk 3]
```diff
@@ -77,6 +77,19 @@ class CWeightedDegreeRBFKernel: public CDotKernel
...
+
+		/** get width
```
Comment (karlnapf) [inline src/shogun/kernel/WeightedDegreeRBFKernel.h]: Same as above

**Comment by karlnapf** [comment]
Thanks! Minor fixes and good to go.

**Comment by lkuchenb** [comment]
I just kept that consistent with the other getter / setter functions in the class. Do you want me to change those, too?

File: src/shogun/kernel/WeightedDegreeRBFKernel.h [hunk 4]
```diff
@@ -77,6 +77,18 @@ class CWeightedDegreeRBFKernel: public CDotKernel
 		virtual EFeatureType get_feature_type() { return F_DREAL; }
 
 
+		/** Set width
+		 *
+		 * @param w new width
+		 */
+		void set_width(float64_t w) { width=w; }
```
Comment (karlnapf) [inline src/shogun/kernel/WeightedDegreeRBFKernel.h]: we should actually assert positivity here, but since this will change in the future, nevermind :)

**Comment by karlnapf** [comment]
It is not high priority to change the others...but if you want to, go for it, cleanups are always good

**Pull request merged**